src/App.jsx

```diff
@@ -41,9 +124,19 @@ class App extends Component {
     firebaseWrapper.logout();
   };
 
-  render() {
-    const { userId } = this.state;
+  makeNotification = (text, color, time, image) => {
```

Comment:
it would be best if this function takes in a type parameter and figures out what image and color it should be from that. We can talk today about what exact kinds of notifications we'll have

Reply:
and we can create some kind of mapping in the form of json between type and color/image